mgcpy/benchmarks/power.py

```diff
@@ -0,0 +1,44 @@
+import numpy as np
+import math
+from mgcpy.independence_tests.dcorr import DCorr
+
+
+def power(independence_test, sample_generator, num_samples=100, num_dimensions=1, repeats=1000, alpha=.05):
```

Comment:
Why pass `sample_generator` as a function? Can't it be imported directly from sims and used?

Reply:
Or does power computation need new data for every repetition.?

Reply:
computing power requires new data for every rep, because each time we need to sample from the alternative distribution

Reply:
passing `sample_generator ` means one needs to specify an alternative distribution (and be able to sample from it) in order to compute power for a test

Reply:
Fine makes sense. Then you should clearly specify the sample_generator's parameters in the docstring.